plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/BaseSqlServerConnectorTest.java

```diff
@@ -385,6 +385,26 @@ public void testNoPushdownOnCaseInsensitiveVarcharColumn()
         }
     }
 
+    @Test
+    public void testNoJoinPushdownOnCaseInsensitiveVarcharColumn()
+    {
+        // if collation on column is caseinsensitive we should not apply join pushdown
+        try (TestTable testTable = new TestTable(
+                onRemoteDatabase(),
+                "test_join_collate",
+                "(collate_column_1 varchar(25) COLLATE Latin1_General_CI_AS, collate_column_2 varchar(25) COLLATE Latin1_General_CI_AS)",
+                List.of("'collation', 'collation'", "'no_collation', 'no_collation'"))) {
+            assertThat(query(format("SELECT n.collate_column_1, n2.collate_column_2 FROM %s n JOIN %s n2 ON n.collate_column_1 = n2.collate_column_1", testTable.getName(), testTable.getName())))
```

Comment:
you can use `%1$s` to reference same variable multiple times in format method

Reply:
Fixed